libr/core/cbin.c

```diff
@@ -2181,9 +2191,13 @@ static int bin_sections(RCore *r, int mode, ut64 laddr, int va, ut64 at, const c
 	} else if (IS_MODE_RAD (mode) && !at) {
 		r_cons_printf ("fs %ss\n", type);
 	} else if (IS_MODE_NORMAL (mode) && !at && !printHere) {
-		r_cons_printf ("[%s]\n", print_segments ? "Segments" : "Sections");
+		r_cons_printf ("[%s]\n"
+                   "Nm Paddr       Size Vaddr      Memsz Perms%s Name\n",
+                   print_segments ? "Segments" : "Sections",
+                   chksum ? chksum : "");
 	} else if (IS_MODE_NORMAL (mode) && printHere) {
-		r_cons_printf ("Current section\n");
+		r_cons_printf ("Current section\n"
```

Comment:
as said, I think this and the previous if should print mostly the same string, so I'd move the string at the beginning of the function and use it in both paths.